src/catkin_pkg/package_version.py

```diff
@@ -140,7 +174,7 @@ def get_forthcoming_label(rst):
             if len(section.children) > 0 and isinstance(section.children[0], docutils.nodes.title):
                 title = section.children[0]
         if title and len(title.children) > 0 and isinstance(title.children[0], docutils.nodes.Text):
-            title_text = title.children[0].rawsource
+            title_text = title.children[0].astext()
```

Comment:
I'm not 100% sure about why this change is needed. I don't know if this was just an uncommon code path (ran into it trying this out on `launch`'s repo), but the docs agree this is the right thing to do:

http://epydoc.sourceforge.net/docutils/private/docutils.nodes.Text-class.html

And without it you get an error saying the `Text` class has no attribute `rawsource`.

Reply:
Related to #308.

Reply:
Ah so this is just another instance of this?

Reply:
I **think** so, but I'm not 100% sure.